howso/utilities/features.py

```diff
@@ -608,8 +608,11 @@ def format_numeric_column(cls, column: pd.Series, feature: Mapping
             return column
 
     @classmethod
-    def format_string_column(cls, column: pd.Series, feature: Mapping
-                             ) -> pd.Series:
+    def format_string_column(
+        cls, 
+        column: pd.Series,
+        feature: Mapping  # noqa: ARG003
```

Comment:
Example fix for keeping an used arg. It'd just be `ARG001`, `ARG002`, etc for the #.